mod/metrics/set/abstract/04_chart.rb

```diff
@@ -77,25 +77,26 @@ def vega_chart
   end
 
   def value_filter_text
-    mv = metric_value_filter
-    return unless mv.present?
-    if mv[:range]
-      "%s < x < %s " % [number_to_human(mv[:range][:from]),
-                       number_to_human(mv[:range][:to])]
+    return "Researched" if filter_hash.empty?
+    value_filter_to_human
+  end
+
+  def value_filter_to_human
+    if filter_hash[:range]
+      value_range_filter_to_human filter_hash[:range]
     else
-      mv[:numeric_value] || mv[:category]
+      f = filter_hash
+      f[:numeric_value] || f[:category] ||
```

Comment:
I know we can't easily have card sets for value types, but I'm thinking we should make some ruby classes for them.  Their logic is very distributed right now.